com.hazelcast.jet.sql.impl.connector.jdbc.JdbcSqlConnectorBounceTest.testQuery [HZ-2292]

_master_ (commit 765cc352c833d7f27d3a438c18a4027b0a3158aa)

Failed on oracle-8: https://jenkins.hazelcast.com/view/Official%20Builds/job/Hazelcast-master-OracleJDK8-nightly/943/testReport/com.hazelcast.jet.sql.impl.connector.jdbc/JdbcSqlConnectorBounceTest/testQuery/
<details><summary>Stacktrace:</summary>

```
com.hazelcast.sql.HazelcastSqlException: 
Encountered "-" at line 1, column 107.
Was expecting one of:
    <EOF> 
    "OBJECT" ...
    "OPTIONS" ...
    ";" ...
    "." ...
    
	at com.hazelcast.sql.impl.CoreQueryUtils.toPublicException(CoreQueryUtils.java:50)
	at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:221)
	at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:172)
	at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:168)
	at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:164)
	at com.hazelcast.sql.SqlService.execute(SqlService.java:89)
	at com.hazelcast.jet.sql.impl.connector.jdbc.JdbcSqlConnectorBounceTest.execute(JdbcSqlConnectorBounceTest.java:109)
	at com.hazelcast.jet.sql.impl.connector.jdbc.JdbcSqlConnectorBounceTest.setUp(JdbcSqlConnectorBounceTest.java:85)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.runners.model.FrameworkMethod$1.runReflectiveCall(FrameworkMethod.java:59)
	at org.junit.internal.runners.model.ReflectiveCallable.run(ReflectiveCallable.java:12)
	at org.junit.runners.model.FrameworkMethod.invokeExplosively(FrameworkMethod.java:56)
	at org.junit.internal.runners.statements.RunBefores.invokeMethod(RunBefores.java:33)
	at org.junit.internal.runners.statements.RunBefores.evaluate(RunBefores.java:24)
	at com.hazelcast.test.AbstractHazelcastClassRunner$ThreadDumpAwareRunAfters.evaluate(AbstractHazelcastClassRunner.java:365)
	at com.hazelcast.test.bounce.BounceMemberRule$BouncingClusterStatement.evaluate(BounceMemberRule.java:304)
	at com.hazelcast.jet.sql.impl.connector.jdbc.DatabaseRule$1.evaluate(DatabaseRule.java:48)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.BlockJUnit4ClassRunner$1.evaluate(BlockJUnit4ClassRunner.java:100)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:366)
	at org.junit.runners.BlockJUnit4ClassRunner.runChild(BlockJUnit4ClassRunner.java:103)
	at com.hazelcast.test.HazelcastSerialClassRunner.runChild(HazelcastSerialClassRunner.java:50)
	at com.hazelcast.test.HazelcastSerialClassRunner.runChild(HazelcastSerialClassRunner.java:29)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at com.hazelcast.test.AfterClassesStatement.evaluate(AfterClassesStatement.java:41)
	at com.hazelcast.test.ChangeLoggingRule$1.evaluate(ChangeLoggingRule.java:44)
	at org.junit.rules.RunRules.evaluate(RunRules.java:20)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.junit.runners.Suite.runChild(Suite.java:128)
	at org.junit.runners.Suite.runChild(Suite.java:27)
	at org.junit.runners.ParentRunner$4.run(ParentRunner.java:331)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:79)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:329)
	at org.junit.runners.ParentRunner.access$100(ParentRunner.java:66)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:293)
	at org.junit.runners.ParentRunner$3.evaluate(ParentRunner.java:306)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:413)
	at org.apache.maven.surefire.junitcore.JUnitCore.run(JUnitCore.java:49)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.createRequestAndRun(JUnitCoreWrapper.java:120)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.executeEager(JUnitCoreWrapper.java:95)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:75)
	at org.apache.maven.surefire.junitcore.JUnitCoreWrapper.execute(JUnitCoreWrapper.java:69)
	at org.apache.maven.surefire.junitcore.JUnitCoreProvider.invoke(JUnitCoreProvider.java:146)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:385)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:507)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:495)
Caused by: com.hazelcast.sql.impl.QueryException: Encountered "-" at line 1, column 107.
Was expecting one of:
    <EOF> 
    "OBJECT" ...
    "OPTIONS" ...
    ";" ...
    "." ...
    
	at com.hazelcast.sql.impl.QueryException.error(QueryException.java:69)
	at com.hazelcast.jet.sql.impl.parse.QueryParser.parse(QueryParser.java:67)
	at com.hazelcast.jet.sql.impl.OptimizerContext.parse(OptimizerContext.java:155)
	at com.hazelcast.jet.sql.impl.CalciteSqlOptimizer.prepare(CalciteSqlOptimizer.java:302)
	at com.hazelcast.sql.impl.SqlServiceImpl.prepare(SqlServiceImpl.java:275)
	at com.hazelcast.sql.impl.SqlServiceImpl.query0(SqlServiceImpl.java:259)
	at com.hazelcast.sql.impl.SqlServiceImpl.execute(SqlServiceImpl.java:204)
	... 54 more
Caused by: org.apache.calcite.sql.parser.SqlParseException: Encountered "-" at line 1, column 107.
Was expecting one of:
    <EOF> 
    "OBJECT" ...
    "OPTIONS" ...
    ";" ...
    "." ...
    
	at com.hazelcast.jet.sql.impl.calcite.parser.HazelcastSqlParser.convertException(HazelcastSqlParser.java:430)
	at com.hazelcast.jet.sql.impl.calcite.parser.HazelcastSqlParser.normalizeException(HazelcastSqlParser.java:183)
	at org.apache.calcite.sql.parser.SqlParser.handleException(SqlParser.java:156)
	at org.apache.calcite.sql.parser.SqlParser.parseStmtList(SqlParser.java:211)
	at com.hazelcast.jet.sql.impl.parse.QueryParser.parse0(QueryParser.java:73)
	at com.hazelcast.jet.sql.impl.parse.QueryParser.parse(QueryParser.java:56)
	... 59 more
Caused by: com.hazelcast.jet.sql.impl.calcite.parser.ParseException: Encountered "-" at line 1, column 107.
Was expecting one of:
    <EOF> 
    "OBJECT" ...
    "OPTIONS" ...
    ";" ...
    "." ...
    
	at com.hazelcast.jet.sql.impl.calcite.parser.HazelcastSqlParser.generateParseException(HazelcastSqlParser.java:42255)
	at com.hazelcast.jet.sql.impl.calcite.parser.HazelcastSqlParser.jj_consume_token(HazelcastSqlParser.java:42066)
	at com.hazelcast.jet.sql.impl.calcite.parser.HazelcastSqlParser.SqlStmtList(HazelcastSqlParser.java:3558)
	at com.hazelcast.jet.sql.impl.calcite.parser.HazelcastSqlParser.parseSqlStmtList(HazelcastSqlParser.java:239)
	at org.apache.calcite.sql.parser.SqlParser.parseStmtList(SqlParser.java:209)
	... 61 more

```
</details>

Standard output can be found here - https://s3.console.aws.amazon.com/s3/buckets/j-artifacts/Hazelcast-master-OracleJDK8-nightly/943/

--- a/hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlConnectorBounceTest.java
+++ b/hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/JdbcSqlConnectorBounceTest.java
@@ -49,7 +49,7 @@ public class JdbcSqlConnectorBounceTest {
 
     private static final int ITEM_COUNT = 5;
     private static final int CONCURRENCY = 10;
-    private static final String TEST_DATABASE_REF = "test-database-ref";
+    private static final String TEST_DATABASE_REF = "test_database_ref";
 
     private String tableName;
 
@@ -88,7 +88,6 @@ public class JdbcSqlConnectorBounceTest {
                         + " name VARCHAR "
                         + ") "
                         + "DATA CONNECTION " + TEST_DATABASE_REF
-                        + "TYPE " + JdbcSqlConnector.TYPE_NAME + ' '
         );
     }
 

